### ISSUE TITLE
explain how to end scenarios properly

### ISSUE THREAD

**Issue by antontimmermans** [scrubbless/MoM2ndTutorial#19]
explain in more detail how to deal with alternate endings and how to end the scenario: losses, wins & investigator dead.
see also 
https://github.com/NPBruce/valkyrie/issues/1070

**Comment by scrubbless** [comment]
Good idea, I'm sure I tackled triggers and endings but didn't explain them in a much detail as I could have.

Also not sure if I explained $end and reviews.
